Pilot1/Uno_IMPROVE/preprocess.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+#########################################################################
+### THIS IS A TEMPLATE FILE. SUBSTITUTE #PATH# WITH THE MODEL EXECUTABLE.
+#########################################################################
+
```

Comment:
Please put here the datasets needed to be able to launch this. Assume CCLE-CCLE etc. that is hard-coded for the example and give paths to exactly download and set this up.

Reply:
Updated. Let me know if this is what you had in mind. Download instructions for the IMPROVE Data are in the README

Reply:
Thanks, after following the steps, I get:
```
> tree IDD
IDD
└── raw_data
    └── gCSI-gCSI
        └── split_0
            ├── test_x_canc.csv
            ├── test_x_drug.csv
            ├── test_y_data.csv
            ├── train_x_canc.csv
            ├── train_x_drug.csv
            ├── train_y_data.csv
            ├── val_x_canc.csv
            ├── val_x_drug.csv
            ├── val_y_data.csv
            ├── x_data_gene_expression_scaler.gz
            └── x_data_mordred_scaler.gz
```

Below is the o/p of preprocess

```

 'x_data_canc_files': [['cancer_gene_expression.tsv', ['Gene_Symbol']]],
 'x_data_dir': 'x_data',
 'x_data_drug_files': [['drug_mordred.tsv']],
 'y_col_name': 'auc',
 'y_data_dir': 'y_data',
 'y_data_files': [['response.tsv']],
 'y_data_preds_suffix': 'predicted',
 'y_data_suffix': 'y_data'}
Traceback (most recent call last):
  File "/Users/mbook/improve-benchmarks/Pilot1/Uno_IMPROVE/uno_preprocess_improve.py", line 463, in <module>
    main(sys.argv[1:])
  File "/Users/mbook/improve-benchmarks/Pilot1/Uno_IMPROVE/uno_preprocess_improve.py", line 455, in main
    ml_data_outdir = run(params)
  File "/Users/mbook/improve-benchmarks/Pilot1/Uno_IMPROVE/uno_preprocess_improve.py", line 202, in run
    params = frm.build_paths(params)
  File "/Users/mbook/IMPROVE/improve/framework.py", line 293, in build_paths
    check_path(x_data_path)
  File "/Users/mbook/IMPROVE/improve/framework.py", line 269, in check_path
    raise Exception(f"ERROR ! {path} not found.\n")
Exception: ERROR ! /Users/mbook/improve-benchmarks/Pilot1/Uno_IMPROVE/IDD/raw_data/x_data not found.

```

Reply:
The output of preprocess looks like an older version I had. Try git pulling.

Also, I think the data download needed a slight adjustment. It's changed it the README as well, but use this to redownload:

wget --cut-dirs=9 -P ~/$IMPROVE_DATA_DIR -np -nH -m https://web.cels.anl.gov/projects/IMPROVE_FTP/candle/public/improve/benchmarks/single_drug_drp/benchmark-data-pilot1/csa_data/